blog/docs/articles/getting-started-blog-p2.md

```diff
@@ -0,0 +1,275 @@
+appropriate license. Since VSCode needs plugins to have IDE features for a language, we recommend that you install the following plugins:
+
+![](/blog/images/getting-started-blog-series/post2/vscode003.png)
+
+![](/blog/images/getting-started-blog-series/post2/vscode004.png)
+
+##### Code Formatting
+
+Go code has a built in style guide and formatter, so naturally you can configure this with both VSCode and JetBrainds IDEs. For VSCode, you can follow the following steps:
+
+![](/blog/images/getting-started-blog-series/post2/vscode005.png)
+
+For JetBrains, you can find more details on how to set it up in [this article](https://www.jetbrains.com/help/idea/integration-with-go-tools.html#gofmt){:target="_blank"}.
+
+## External Setup
+
+Now that you have a working local setup, there are some external things which you will have to set up as well. These can be grouped by how to join the community, how to claim
+issues, and how to make a PR.
+
+### Community
+
+As you are working on your code, you will probably want to discuss your changes with others and ask questions. Knative uses __Slack__ as the primary means of communication,
+along with __a mailing list__. You can find links to these communication channels [here](https://knative.dev/docs/community/#communication-channels){:target="_blank"}. There are also weekly 
+and bi-weekly working group meetings which you can join on zoom where we have discussions. To find the community calendar(which includes the working group meeting times and
+links), as well as the link to the slack and the mailing list, you can [read this article](https://knative.dev/docs/community/#meetings){:target="_blank"}.
+
+![](/blog/images/getting-started-blog-series/post2/doodle002.png)
+
+### Claiming an Issue
+
+Now that you have a working local set up, you probably want to tackle an issue! To claim an issue, go to the "Issues" tab of the repository you want to contribute to,
+and then look through the issues. If this is one of your first contributions to that repository, you should look for issues labelled "good-first-issue". To filter the
+issues by a particular label, follow the steps [outlined here](https://docs.github.com/en/github-ae@latest/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests#filtering-issues-and-pull-requests-by-labels){:target="_blank"}.
+If there are no issues labelled "good-first-issue", you can always ask on [Slack](https://cloud-native.slack.com/archives/C04LN0620E8){:target="_blank"} if there are any good issues for you.
+Additionally, if the issues labelled "good-first-issue" are all taken then you can try commenting on them and seeing if they are still actively being worked on. Once
+you have an issue you would like to work on, comment "/assign" and the Prow bot will assign it to you (more on Prow in a bit!).
```

Comment:
Not sure if anybody can assign tickets. I guess you have to be Knative GH org member.

I would write something like:
- Tell that you want to work on this issue
- Don't ask for permission though

Reply:
@knative/productivity-leads can anyone run the `/assign` command on Knative issues?

Reply:
yes